docs/content/docs/recipes/react.md

```diff
@@ -120,60 +83,64 @@ const azureClientConfig = {
 1. Replace `TODO 1` with the following code. Note that there is only one object in the container: a SharedMap holding the timestamp. Note also that `sharedTimestamp` is the ID of the `SharedMap` object and it must be unique within the container.
 
     ```js
-    const { containerId, isNew } = getContainerId();
-    const azureClient = new AzureClient(azureClientConfig);
```

Comment:
I'm still not sure I fully understand the motivation for removing `AzureClient` from all of these recipes/examples. It was specifically created to provide an easy path for users to connect both with Tinylicious and Azure using only a configuration change. Using `TinyliciousClient` everywhere means that users will have to change clients and packages whenever they migrate to any deployment or even to connect to any other remote clients. This feels like we're unnecessarily adding in steps that a developer will need to do later on and "re-learn" concepts for two clients, instead of just one that can cater to both their dev and production scenarios

Reply:
Ok, had a discussion regarding the role of Azure vs Tinylicious client in our examples with @skylerjokiel . I'm okay with removing it from our tutorial documentation here to keep it non-Azure focused and have that be in the Azure docs instead. However, in our FluidExamples repo, let's have one example, Brainstorm, that stays on the `AzureClient` so that devs have an option to reference when building a full app